<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
    <class>ChangePasswordDialog</class>
    <widget class="QDialog" name="ChangePasswordDialog">
        <property name="geometry">
            <rect>
                <x>0</x>
                <y>0</y>
                <width>400</width>
                <height>250</height>
            </rect>
        </property>
        <property name="windowTitle">
            <string>Change Developer Password</string>
        </property>
        <property name="modal">
            <bool>true</bool>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
            <item>
                <widget class="QLabel" name="titleLabel">
                    <property name="font">
                        <font>
                            <pointsize>12</pointsize>
                            <weight>75</weight>
                            <bold>true</bold>
                        </font>
                    </property>
                    <property name="text">
                        <string>Change Password</string>
                    </property>
                    <property name="alignment">
                        <set>Qt::AlignCenter</set>
                    </property>
                </widget>
            </item>
            <item>
                <layout class="QFormLayout" name="formLayout">
                    <item row="0" column="0">
                        <widget class="QLabel" name="oldPasswordLabel">
                            <property name="text">
                                <string>Old Password:</string>
                            </property>
                        </widget>
                    </item>
                    <item row="0" column="1">
                        <widget class="QLineEdit" name="oldPasswordLineEdit">
                            <property name="echoMode">
                                <enum>QLineEdit::Password</enum>
                            </property>
                        </widget>
                    </item>
                    <item row="1" column="0">
                        <widget class="QLabel" name="newPasswordLabel">
                            <property name="text">
                                <string>New Password:</string>
                            </property>
                        </widget>
                    </item>
                    <item row="1" column="1">
                        <widget class="QLineEdit" name="newPasswordLineEdit">
                            <property name="echoMode">
                                <enum>QLineEdit::Password</enum>
                            </property>
                        </widget>
                    </item>
                    <item row="2" column="0">
                        <widget class="QLabel" name="confirmPasswordLabel">
                            <property name="text">
                                <string>Confirm New Password:</string>
                            </property>
                        </widget>
                    </item>
                    <item row="2" column="1">
                        <widget class="QLineEdit" name="confirmPasswordLineEdit">
                            <property name="echoMode">
                                <enum>QLineEdit::Password</enum>
                            </property>
                        </widget>
                    </item>
                </layout>
            </item>
            <item>
                <layout class="QHBoxLayout" name="horizontalLayout">
                    <item>
                        <spacer name="horizontalSpacer">
                            <property name="orientation">
                                <enum>Qt::Horizontal</enum>
                            </property>
                            <property name="sizeHint" stdset="0">
                                <size>
                                    <width>40</width>
                                    <height>20</height>
                                </size>
                            </property>
                        </spacer>
                    </item>
                    <item>
                        <widget class="QPushButton" name="saveButton">
                            <property name="text">
                                <string>Save</string>
                            </property>
                            <property name="default">
                                <bool>true</bool>
                            </property>
                        </widget>
                    </item>
                    <item>
                        <widget class="QPushButton" name="cancelButton">
                            <property name="text">
                                <string>Cancel</string>
                            </property>
                        </widget>
                    </item>
                </layout>
            </item>
        </layout>
    </widget>
    <resources/>
    <connections/>
</ui>